### ISSUE TITLE
gocredits integration

### ISSUE THREAD

**Issue by syucream** [reproio/columnify#4]
https://songmu.jp/riji/entry/2019-04-16-gocredits.html

**Comment by t2y** [comment]
I confirmed that recent gocredits has `-skip-missiong` option. How about ignoring as a workaround?

```bash
$ gocredits -version
gocredits v0.2.0 (rev:HEAD)
$ gocredits -skip-missing -w .
could not find the license for "github.com/apache/arrow/go/arrow"
$ wc -l CREDITS 
4681 CREDITS
```

**Comment by t2y** [comment]
I merged #64. I don't know how to release columnify, so I would like to ask someone to release it if needed.

**Comment by syucream** [comment]
@t2y Just create a new GitHub release on this repo :) 👉  https://github.com/reproio/columnify#release
